code/modules/clothing/under/accessories/storage.dm

```diff
@@ -73,6 +87,15 @@
 	icon_state = "unathiharness2"
 	_color = "unathiharness2"
 	slots = 2
+	can_only_hold = list("/obj/item/weapon/hatchet", "/obj/item/weapon/kitchen/utensil/knife")
+
+/obj/item/clothing/accessory/storage/bandolier
```

Comment:
Whoops.
Didn't notice that because `knifeharness` is the only one with custom code